ingress/contour/central-ingress.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: projectcontour.io/v1
+kind: HTTPProxy
```

Comment:
Is that right for TCP passthrough? On other providers like traefik I've seen `kind: IngressRoute` or `kind: IngressRouteTCP` separated out. I'll take your word for it I guess 🤷

Reply:
It works. This is an enhancement from the old IngressRoute object where you dont need any routes anymore

Reply:
https://projectcontour.io/from-ingressroute-to-httpproxy/